app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def add_room
+    @guest = Guest.find(params[:id])
+    room = Room.find(params[:room_id])
+    @guest.rooms << room
+    redirect_to "/guests/#{@guest.id}"
```

Comment:
The logic in this method is sound, we just want to make sure we put it in a RESTful controller action like "create".